examples/coca/README.md

```diff
@@ -0,0 +1,55 @@
+# CoCa: Contrastive Captioners are Image-Text Foundation Models
+
+## Introduction
+
+Coca is a a minimalist design to pretrain an image-text encoder-decoder foundation model jointly with contrastive loss and captioning loss, thereby subsuming model capabilities from contrastive approaches like CLIP and generative methods like SimVLM.
+In contrast to standard encoder-decoder transformers where all decoder layers attend to encoder outputs, CoCa omits cross-attention in the first half of decoder layers to encode unimodal text representations, and cascades the remaining decoder layers which cross-attend to the image encoder for multimodal image-text representations.
+The structure of DiT model and DiT blocks is shown below[<a href="#references">1</a>].
+
+<p align="center">
+  <img width="550" alt="figure" src="https://github.com/mlfoundations/open_clip/assets/52945530/ca75eccb-ea1f-4e9a-b628-1b88b691b849">
+</p>
+<p align="center">
+  <em> Figure 1. Detailed illustration of CoCa architecture and training objectives. [<a href="#references">1</a>] </em>
+</p>
+
+## Get Started
+In this tutorial, we will introduce how to run inference and finetuning experiments using MindONE.
```

Comment:
The readme only contains inference examples, not "inference and finetuning experiments".